test/Common/TestOrchestratorClient.cs

```diff
@@ -155,7 +155,7 @@ public async Task<DurableOrchestrationStatus> WaitForCustomStatusAsync(TimeSpan
             while (sw.Elapsed < timeout);
 
             output.WriteLine($"Timeout expired. {status.Name} ({status.InstanceId}) currently shows a status of '{status.CustomStatus}'.");
-            Assert.Equal(expectedStatus, status.CustomStatus);
+            Assert.Equal(expectedStatus.ToString(), status.CustomStatus?.ToString());
```

Comment:
Is this just to give more clear output in the case of a failure?

Reply:
Yeah, it wasn't actually printing the JValue values for some reason, so I switched it to `ToString()`.